tests/src/test/scala/com/stripe/agate/eval/OperationTest.scala

```diff
@@ -252,4 +252,16 @@ object OperationTest extends Properties("OperationTest") {
 
     c0 && c1 && c2 && ex1 && ex2
   }
+
+  property("NonZero works") = {
+    // Check that the NonZero operator works. Note that more detailed tests are in the TensorTest file
+
+    val c1 = {
+      val t0 = tensor"[[3, 4, 0, 6], [0, 0, 0, 7]]"
+      val expected = tensor"[[0, 0, 0, 1], [0, 1, 3, 3]]".map(DataType.Int64)(_.toLong)
+      test("NonZero", t0 :: Nil, List(expected), Map.empty)
+    }
+
+    c1
```

Comment:
I don’t think we need to capture c1 and then return it. Can we just remove the val c1 and return test?

We used that pattern in other places when there were multiple claims. If we left some around with only 1 it is because e changed the code and got lazy about cleaning it up after we removed claims.